build/images/Dockerfile.build.coverage

```diff
@@ -24,7 +24,9 @@ RUN go mod download
 
 COPY . /antrea
 
-RUN make antrea-agent antrea-controller antrea-cni antctl-linux antrea-controller-instr-binary antrea-agent-instr-binary antctl-instr-binary
+# To make the GO binaries portable, so that they can run on OS,
+# other from the one on which it was built we need to disable CGO.
```

Comment:
This comment does not provide enough information. Because this is a Docker build, usually there is no need to disable CGO. So it should explicitly mention that this change is needed because the e2e tests for antctl currently copy the antctl binary from the Docker container to the host Node, and run the binary from the host. The host may be missing some system libraries that are required to run the binary, so we need to disable CGO to avoid such dependencies and ensure the binary is portable.